x-pack/test/functional/services/observability/alerts.ts

```diff
@@ -54,7 +55,7 @@ export function ObservabilityAlertsProvider({ getPageObjects, getService }: FtrP
   };
 
   const clearQueryBar = async () => {
-    return await (await getQueryBar()).clearValueWithKeyboard({ charByChar: true });
+    return await (await getQueryBar()).clearValueWithKeyboard();
```

Comment:
I actually tried to use `clearValue` here rather than `clearWithKeyboard`, it should work as it's a `textarea` we're targeting, but for some reason it wouldn't work 🤷‍♀️